test/parallel/test-bootstrap-modules.js

```diff
@@ -10,7 +10,7 @@ const assert = require('assert');
 
 const isMainThread = common.isMainThread;
 const kCoverageModuleCount = process.env.NODE_V8_COVERAGE ? 1 : 0;
-const kMaxModuleCount = (isMainThread ? 65 : 87) + kCoverageModuleCount;
+const kMaxModuleCount = (isMainThread ? 65 : 88) + kCoverageModuleCount;
```

Comment:
I currently have a PR open (https://github.com/nodejs/node/pull/26531) to refactor this test so we're no longer relying on a vague ever increasing number and instead make conscious decisions about whether modules should be loaded by the bootstrapping process (when it fails, the refactored test tells you what modules were loaded that weren't expected to be (i.e. count would have gone up) or which modules were expected to be loaded but weren't (i.e. count would have gone down (but we're rarely done that))).

Anyway this is more of a note that if this PR lands before https://github.com/nodejs/node/pull/26531 I'll need to adjust, or likewise if https://github.com/nodejs/node/pull/26531 lands first then this PR will need to be adjusted.

Reply:
FWIW, we don't really need to load heap_utils during the bootstrap of workers, it can be done lazily. (Or internal/error-serdes can lazily load v8)